.github/workflows/wheels.yml

```diff
@@ -0,0 +1,54 @@
+name: Build wheels
+
+on: [push]
+# workflow_dispatch:{}
+
+env:
+  DEBIAN_FRONTEND: noninteractive
+  FORCE_CUDA: 1
+  # all values:
+  # 3.5;3.7;5.0;5.2;5.3;6.0;6.1;6.2;7.0;7.2;7.5;8.0;8.6
+  # with optional +PTX
+  # https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/packaging/pkg_helpers.bash#L63
+  # you need at least cuda 5.0 for some of the stuff compiled here.
+  TORCH_CUDA_ARCH_LIST: "6.0 6.1 7.0 7.5 8.6"
+
+jobs:
+  build_wheels:
+    strategy:
+      matrix:
+        # python: ["3.7", "3.8", "3.9", "3.10"]
+        python: ["3.10"]
```

Comment:
let's support python `3.9` and `3.10` to be coherent with the [conda builds](https://anaconda.org/xformers/xformers/files)

Reply:
ok!